benchmarks/viscoelastic_plastic_shear_bands/gerya_2019/gerya_2019_vp.prm

```diff
@@ -0,0 +1,196 @@
+
+# Solver parameters
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Stokes solver type = block AMG
+    set Linear solver tolerance = 1e-7
+    set Number of cheap Stokes solver steps = 200
+  end
+  subsection Newton solver parameters
+    set Max Newton line search iterations        = 0
+    set Max pre-Newton nonlinear iterations      = 500 # Ensures only defect-correction Picard method is used
+    set Maximum linear Stokes solver tolerance   = 1e-2
+    set Nonlinear Newton solver switch tolerance = 1e-5
+    set SPD safety factor                        = 0.9
+    set Stabilization preconditioner             = SPD
+    set Stabilization velocity block             = SPD
+    set Use Newton failsafe                      = false
+    set Use Newton residual scaling method       = false
+    set Use Eisenstat Walker method for Picard iterations = true
+  end
+end
+
+
+# Model geometry (100x100 km, initial 50 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 2
+    set Y repetitions = 2
+    set X extent      = 100e3
+    set Y extent      = 100e3
+  end
+end
+
+# Globally refine the mesh. A minimum of 3 global refinements
+# is needed to resolve the edges of the weak viscous inclusion.
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 5
+  set Time steps between mesh refinement = 0
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree     = 2
+  set Stokes velocity polynomial degree = 2
+  set Temperature polynomial degree     = 1
+end
+
+# Velocity boundary conditions (5e-9 m/s on each boundary)
+# The imposed velocity produces a background strain-rate
+# of 10e-13 1/s. 
+subsection Boundary velocity model
+  set Prescribed velocity boundary indicators = left x: function, right x:function, top y: function, bottom y: function
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = vel=0.15778463
+    set Function expression = if (x<50e3, vel, -vel); if (y<50e3, -vel, vel);
+  end
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 3
+  set Names of fields  = block, air, inclusion
+end
+
+# Spatial domain of different compositional fields
+subsection Initial composition model
+  set Model name = function
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  =
+    set Function expression = if ( ( x<43.75e3 && y>25.e3 && y<75.e3) || (x>56.25e3 && y>25.e3 && y<75.e3) || (y>56.25e3 && y<75.e3 && x>=43.75e3 && x<=56.25e3) || (y<43.75e3 && y>25.e3 && x>=43.75e3 && x<=56.25e3), 1, 0); \
+                              if (y<=25.e3 || y>=75.e3, 1, 0); \
+                              if (y<=56.25e3 && y>=43.75e3 && x>=43.75e3 && x<=56.25e3, 1, 0);  
+  end
+end
+
+# Composition boundary conditions
```

Comment:
I don't think you need this whole section if you don't actually have any boundary conditions for the composition.

Reply:
Yep, this could be deleted. However, I like to keep this section in as a reminder that the compositional fields in fact have no boundary conditions. Purely there to avoid confusion (on my end) later on :)